views/index.pug

```diff
@@ -19,35 +19,35 @@ html
       | : Returns a list of all Pokémon Go Battle leagues.
     h1
       span.get GET
-      | /leagues/:league/leads
+      | /leagues/:leagueSlug/leads
     code
       span.desc Method
       | : 
       span.get GET<br>
       span.desc Route
-      | : https://placeholder.url/leagues/:league/leads<br>
+      | : https://placeholder.url/leagues/:leagueSlug/leads<br>
```

Comment:
I had to guess league slug since the the leagues route only returns names, not slugs

![image](https://user-images.githubusercontent.com/5775716/83801964-f9c55680-a677-11ea-9f0a-0f0ec56ab1f7.png)